sources/packages/web/src/views/institution/InstitutionUserProfile.vue

```diff
@@ -1,7 +1,11 @@
 <template>
   <full-page-container>
     <template #header>
-      <header-navigator title="Institution" subTitle="My Profile" />
+      <header-navigator
+        data-cy="myProfileHeader"
```

Comment:
institutionUserProfileHader may be more suitable for data-cy

Reply:
Agreed, updated it